Catalog/Services/Comments/CommentsRepository.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Catalog.Services.Comments.Interfaces;
+using Common.Interfaces;
+using Common.Models.Comments;
+using Common.Validatiors.Comments;
+using FluentValidation;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Catalog.Services.Comments
+{
+    public class CommentsRepository : ICommentsRepository
+    {
+        public readonly IDataAccessor<Comment> Comments;
+
+        public CommentsRepository(IDataAccessor<Comment> dataAccessor)
+        {
+            Comments = dataAccessor;
+        }
+
+        public async Task<bool> AddComment(Comment comment)
+        {
+            return await Comments.Add(comment) == 1;
+        }
+       
+        public async Task<bool> DeleteComment(int id)
+        {
+            var validator = new CommentIdValidator();
+            validator.ValidateAndThrow(id);
+
+            return await Comments.Delete(id) == 1;
```

Comment:
The idea of making an assumption and return its result immediately seems more minimalistic and takes fewer lines of code to get the result, but it is harder to debug later on, because we can't just hover over it and get the value, but should go to the locals, which is confusing if you have a lot of different variables at the same time. This is not a problem or blocker though - just expressing my ideas. I am ok with this, and I don't insist on nay kind of changes for it so far.